src/desktop/osx/CMakeLists.txt

```diff
@@ -1,37 +1,19 @@
 # Desktop Integration
-# Copyright (c) 2022  Igara Studio S.A.
+# Copyright (c) 2022-2025  Igara Studio S.A.
 
-find_library(QUARTZ_LIBRARY Quartz)
+find_library(QUICKLOOK_LIBRARY QuickLookThumbnailing)
 
-add_library(AsepriteThumbnailer SHARED
-  main.mm
+set(extension_target AsepriteThumbnailer)
```

Comment:
Probably we keep the AsepriteThumbnailer name (without incorporating the `extension_target` variable).